In [1]:
import pandas as pd
import numpy as np

# Marotta e gli stadi in Italia

> Questa è la nota dolente, i numeri contano per fare delle riflessioni. Negli ultimi 20 anni, in Europa, sono stati costruiti 250 nuovi stadi, di questi solo 5 in Italia. Questo la dice lunga su quanto siamo fanalino di coda

Questa e' una frase riportata da alcuni siti e si riferisce a una dichiarazione del dirigente dell'Inter Marotta. Siamo convinti che la maggior parte delle persone che hanno ascoltato o letto questa frase, la possano trovare lapalissianamente vera o comunque verosimile. Marotta, usando determinati numeri, evidenzia quella che, secondo lui, e' una chiara carenza di nuovi stadi in Italia negli ultimi venti anni. 

Quando si usano i numeri per fare delle dichiarazioni, come ha fatto Marotta, ci si puo' chiedere se tale affermazione abbia valore dal punto di vista statistico. Per divertimento (persone noiose come noi trovano divertenti queste cose. Se voi non lo trovate divertenti siete persone assolutamente normali) cerchiamo di capire se i numeri da lui citati la dicano lunga veramente.

Da un punto di vista statistico, quindi, facciamo un [test di verifica d'ipotesi](https://it.wikipedia.org/wiki/Test_di_verifica_d%27ipotesi). Di solito questo test prevede un'ipotesi nulla $H_{0}$ e una alternativa $H_{a}$ e, in genere, si cerca di rigettare l'ipotesi nulla a favore di quella alternativa. Senza entrare troppo nei dettagli, si stabilisce un livello di significativita' statistica (in genere il $5%$) e, attraverso i dati raccolti, lo si confronta con il cosidetto *p-value*: se tale valore e' minore del livello di significativita', si rigetta l'ipotesi nulla, altrimenti si fallisce a rigettarla. L'argomento di questo articolo non e' spiegare come funziona un test di verifica d'ipotesi, e la spiegazione appena data e' molto rozza e non tiene conto di tante raffinatezze sul significato di prova, di *p-value* e tanto altro. Qui stiamo a divertirci (anche sulla definizione di divertimento non vogliamo perdere troppo tempo), e stante lo scopo definiamo l'ipotesi nulla:

$$
H_{0} : \text{non c'e' differenza tra il numero di stadi nuovi costruiti negli ultimi 20 anni in Europa e l'Italia} 
$$

Quella alternativa (l'ipotesi Marotta)

$$
H_{a} : \text{C'e' differenza tra il numero di nuovi stadi costruiti in Europa e quelli costruiti in Italia.} 
$$

Per quanto riguarda $H_{a}$, Marotta va oltre e dichiara che l'Italia e' fanalino di coda, e questo ci aiuta anche per quanto riguarda il verso, o la coda della distribuzione, della disuguaglianza che andremo a studiare.  

## Livello 1: 250, 5, 20 e l'Europa

Il primo livello di analisi utilizza, come informazione, solo quella contenuta nella frase di Marotta, ossia 250 stadi in 20 anni in Europa e 5 in Italia. Se l'intento di Marotta e' quello di concludere che in Italia si costruiscono pochi stadi, allora possiamo calcolare se cinque nuovi stadi in Italia, rispetto ai rimanenti 245 nel resto d'Europa, siano, in maniera significativa, statisticamente molto al di sotto della media Europea. 

Dato che Marotta parla in generale di Europa, consideriamo prima tre diverse ipotesi su cosa intendesse per Europa. Dato che non specifica a cosa si possa riferire quando parla di Europa, prendiamo come prima istanza 3 diverse definizioni di Europa per calcolare quante nazioni, in totale, sono nel continente:
- Europa come EU
- Europa come continente
- Europa come UEFA

A seconda delle definizioni di Europa, cambia il numero di nazioni che ne fanno parte e questo influenza la media di nuovi stadi per ogni paese: 

$$
\overline{nuovi\_stadi} = \frac{250}{k}
$$

dove $k$ rappresenta il numero di nazioni della definizione di Europa. 

### 1.a. Europa come EU

Nel caso in cui vogliamo considerare Europa come gli attuali stati che formano l'Unione Europea, allora $k = 27$ e quindi avremo una media attesa di:

$$
\mu = \frac{250}{27} \approx 9.26
$$

In [2]:
print(f"mu = {np.round(250/27, 2)}")

mu = 9.26


L'Italia, a quanto dice Marotta, negli ultimi 20 anni ha costruito 5 stadi, quindi la differenza con la media e' di:

$$
x - \mu = 5 -9.26 = -4.26
$$

e quindi, in Italia, ha circa 4.26 in meno rispetto alla media Europea. E' abbastanza per dire che siamo sotto la media europea? Supponiamo che il numero di stadi per ogni nazione europea segua un modello di Poisson (ossia, esiste la stessa probabilita' di costruire nuovi stadi in ogni nazione in maniera indipendente), quindi ogni nuovo stadio ha probabilita' 

$$
p = \frac{1}{k} = \frac{1}{27}
$$

di essere costruito in quella nazione, allora il numero di stadi per ogni nazione risulta:

$$
X \sim \text{Binomial}(n = 250, p = 1/27)
$$

E dato che $n$ e' grande mentre $p$ e' relativamente piccolo, allora possiamo usare un'approssimazione di Poisson:

$$
X \sim Poisson(\lambda = np = 9.26)
$$

Possiamo ora rispondere alla domanda: qual'e' la probabilita' di osservare la costruzione di 5 o meno di 5 stadi:

$$
P( X \le 5 | \lambda = 9.26)
$$

In [3]:
from scipy.stats import poisson

n = 250
p = 1/27
lambda_ = n*p
p_value = poisson.cdf(5, lambda_)
print(f"the resulting p-value is {p_value}")

the resulting p-value is 0.10082956070176977


Abbiamo quindi un p-value di circa 0.101 e quindi possiamo concludere che: 
> Considerando il numero di nazioni all'interno dell'Europa come le nazioni dell'Unione Europea e sotto l'ipotesi di un modello dove ogni nazione ha la stessa probabilita' di costruire un nuovo stadio negli ultimi 20 anni, il fatto che l'Italia ne abbia costruiti di meno non e' statisticamente significativo, ossia non e' inusuale

Riassumendo:
- media attesa: 9.26
- media osservata: 5
- differenza: -4.26
- z-score (sotto approssimazione di Poisson):
$$
z = \frac{5-9.26}{\sqrt{9,26}} = -1.4
$$
- p-value: 0.102

E quindi l'Italia e' sotto la media, ma non in maniera significativa (ossia, dal punto di vista strettamente probabilistico, la spiegazione puo' essere semplicemente data dal caso)

### 1.b., 1.c. Europa Continentale e UEFA

Se parliamo di calcio, e' difficile che ci riferiamo all'Europa considerando solo i paesi che fanno parte dell'Unione Europea, correndo il rischio di non considerare nazioni come l'Inghilterra che, al suo interno, contiene quattro importanti federazione UEFA quali Scozia, Galles, Nord Irlanda e Inghilterra. Usando la stessa metodologia usata nel caso 1.a. 

Come valore di $k$ useremo 44 come Europa Continentale e 55 come UEFA. Usando questi parametri otteniamo

In [4]:
import pandas as pd

n_countries = [44, 55]
x = 5
mu = []
ita_diff = []
z_score = []
p_value = []

for k in n_countries:
    mu.append(250/k)
    ita_diff.append(x - 250/k)
    z_score.append((x - 250/k) / np.sqrt(250/k))
    p_value.append(poisson.cdf(x, 250/k))

pd.DataFrame(
    {
        "Europa vista come": ['Europa Continentale', 'UEFA'],
        "Numero Nazioni": n_countries,
        "Media": mu,
        "Differenza": ita_diff,
        "z_score": z_score,
        "p_value": p_value
    }
)

,Europa vista come,Numero Nazioni,Media,Differenza,z_score,p_value
0,Europa Continentale,44,5.681818,-0.681818,-0.286039,0.498039
1,UEFA,55,4.545455,0.454545,0.213201,0.695147


Come ci si poteva aspettare, aumentando il numero di nazioni e tenendo costante il numero di nuovi stadi, diminuisce la media per nazione e quindi anche la differenza con il valore osservado di 5 per i nuovi stadi italiani. Di conseguenza, l'affermazione che il numero di nuovi stadi in Italia sia molto minore di quello del resto d'Europa perde di significato statistico. 

Volendo usare un linguaggio statistico un po' piu' rigoroso, se consideriamo 2 ipotesi $H_{0}$ (Null Hypothesis) e $H_{A}$ (Alternative Hypothesis) tali che 
- $H_{0} : \text{L'Italia ha lo stesso numero di nuovi stadi costruiti negli ultimi 20 anni rispetto all'Europa}$:
- $H_{A} : \text{L'Italia ha un numero di nuovi stadi costruiti negli ultimi 20 anni minore rispetto all'Europa}$:

I valori dei p-value ci dicono che non riusciamo a rigettare l'ipotesi nulla e che i dati ci dicono che non esiste una differenza statisticamente significativa tra il valore osservato Italiano e la media Europea nei tre casi considerando le nazioni Europee come a. Solo EU; b. Europa Continentale; c. UEFA.

## Livello 2: Europa, GDP e numero di nuovi stadi

Il Livello 1 considerava la probabilita' di costruire nuovi stadi equamente distribuita lungo tutte le nazioni europee, senza fare nessuna distinzione. Nel Livello 2, proviamo a vedere se inserendo il GDP pro capite nel nostro modello, il fatto che l'Italia abbia costruito solo 5 stadi rispetto ai 245 nel resto d'Europa ci porta a concludere che il paese della pizza, il mandolino, Leonardo Da Vinci, Sinner e del Festival di Sanremo sia in ritardo rispetto alla questione stadi rispetto al continente responsabile di tutte le guerre, carestie, crisi economiche, Trump, le cavallette della storia moderna ed antica. 

### Framework Teorico

Il modello che andremo a considerare per livello 2 ha, come caratteristica principale, l'ipotesi che il numero di nuovi stadi costruito nell'arco degli ultimi 20 anni dipenda dal GDP (o prodotto interno lordo) *pro capite* della nazione dove si costruiscono gli stadi. Dal punto di vista pratico, diciamo che piu' un paese e' ricco, piu' la probabilita' di costruire nuovi stadi sia maggiore rispetto ai paesi con un reddito pro capite inferiore. Per rendere le cose piu' semplici, faremo qualche ulteriore semplificazione:
- Anche se abbiami i dati del GDP pro capite degli ultimi 20 anni, andremo ad usare il dato del solo 2023 (dato consolidato rispetto alle stime per gli anni 2024-25)
- Considerando le nazioni incluse nella lista UEFA, assegneremo all'Inghilterra, Scozia, Galles e Irlanda del Nord il GDP pro capite dell'intero Regno Unito 

Fatte queste precisazioni, possiamo modellare il numero di nuovi stadi per paese come una variabile stocastica con un valore atteso (valore  d'aspettazione o *Expected Value*) proporzionale al GDP pro capite nazionale. Ci sono due modi per costruire questo modello, uno piu' semplice che esegue un'allocazione proporzionale per avere una baseline di confronto tra l'Italia e il resto del mondo, e uno piu' preciso che e' quello di regressione Poissoniana. Con quest'ultimo, possiamo considerare, come nel caso precedente, un *p-value* per decidere se, statisticamente, il numero di 5 nuovi stadi e' significamente diverso (maggiore o minore) rispetto all'Italia.

#### Livello 2 - caso EU, Modello Proporzionale Probabilistico 

Definiamo
- $i \in {1, \dots, N}$ indici dei paesi
- $g_{i}$ e' il GDP *pro capite* del paese $i$
- $S = 250$ e' il numero totale di stadi costruiti in Europa negli ultimi 20 anni
- $p_{i} = \frac{g_{i}}{\sum_{j}g_{j}}$ frazione del GDP *pro capite* per il paese $i$ 

Allora il numero di stadi atteso per ogni paese $i$ sotto le ipotesi di proporzionalita' e':

$$
\mathbb{E}[{Y_{i}}] = S \cdot \frac{g_{i}}{\sum_{j=1}^{N}g_{j}}
$$

e da un un benchmark deterministico. Se, quindi, $y_{i} = 5$ allora possiamo definire che:

- Sopra la media: $y_{i} > \mathbb{E}[{Y_{i}}]$
- In media: $y_{i} \approx \mathbb{E}[{Y_{i}}]$ (entro una determinata tolleranza)
- Sotto la media: $y_{i} < \mathbb{E}[{Y_{i}}]$

Ma con questo modello non c'e' una misura di significanza statistica, ossia non possiamo decidere se, dal punto di vista statistico, valori sopra o sotto la media siano dettati dal caso oppure no.

Vediamo con i valori che estraiamo dai dati, quale puo' essere il nostro benchmark nel caso di ritenere l'Europa come costituita dai soli paesi EU (caso Satana).

In [5]:
from pathlib import Path

data_path = Path('data/processed/EU_countries_GDP.pkl')
EU_countries_GDP = pd.read_pickle(data_path)

# total number of stadiums in the last 20 years

TOTAL_STADIUMS = 250

# Compute expected stadiums proportional to GDP per capita
EU_countries_GDP["GDP_share"] = EU_countries_GDP["GDP"] / EU_countries_GDP["GDP"].sum()
EU_countries_GDP["expected_stadiums"] = TOTAL_STADIUMS * EU_countries_GDP["GDP_share"]

EU_countries_GDP.head()

,Country,Population,GDP,GDP_share,expected_stadiums
0,Austria,9.1,56579.504175,0.048672,12.167905
1,Belgium,11.6,55291.475454,0.047564,11.890903
2,Bulgaria,6.5,15853.208637,0.013637,3.409368
3,Croatia,3.9,22184.227930,0.019084,4.770907
4,Cyprus,1.3,36623.691406,0.031505,7.876237


In [6]:
EU_countries_GDP[EU_countries_GDP['Country'] == 'Italy']

,Country,Population,GDP,GDP_share,expected_stadiums
14,Italy,59.2,39277.083878,0.033787,8.446872


Quindi nel caso in cui teniamo in considerazione solo i paesi dell'Unione Europea e il loro GDP *pro capite* e supponendo che negli ultimi 20 anni sono stati costruiti 250 stadi nell'EU, ci aspetteremmo circa 8.4 stadi nuovi in Italia. Confrontando il valore atteso (8.4) con quello osservato (5), possiamo dire che il valore osservato e' al di sotto delle aspettative (mancano quasi 3 stadi e mezzo. 3 stadi, una curva, e un distinto superiore o inferiore).

Questo modello, pero', non ci dice se questa discrepanza e' statisticamente significativa, ossia se potrebbe essere dettata dal caso (fluttuazione statistica) oppure c'e' qualche potere forte che cospira contro la costruzione di nuovi stadi (in questo caso potremmo addirittura parlare di massoni che si oppongono a costruizioni, una lotta fratricida insomma). 

Per avere un quadro statisticamente piu' preciso, utilizziamo un modello di Poisson (modello stocastico con inferenza statistica).

#### Livello 2: Caso Satana (EU), Modello Stocastico Poissoniano

Il modello poissoniano e' particolarmente adatto per valutare il numero di eventi in un determinato intervallo temporale (si usa una binomiale negativa o di Pascal se e' piu' dispersa, ossia ha una varianza maggiore).

In questo caso supponiamo che il numero di stadi nel tempo e' modellato attraverso una distribuzione di Poisson:

$$
Y_{i} = \text{Poisson}(\lambda_{i})
$$

con 
- $\lambda_{i} = \alpha \cdot g_{i}$

Per calcolare $\alpha$ usiamo il fatto che il numero totale di stadi negli ultimi 20 anni e' stato, secondo Marotta, 250, per cui:

$$
\sum_{i} \lambda_{i} = 250 = \alpha \sum_{i} g_{i}
$$

con, ancora, $g_{i}$ il GDP *pro capite* del paese $i$. Quindi la stima di $\alpha$ e'

$$
\alpha = \frac{250}{\sum_{i} g_{i}}
$$

e quindi

$$
\lambda_{i} = 250 \cdot \frac{g_{i}}{\sum_{j}g_{j}}
$$

I lettori piu' attenti si saranno accorti che il risultato finale e' uguale al modello deterministico, ma, in questo caso, abbiamo una ***distribuzione*** che ci permette una valutazione statistica. 

#### Hypothesis Test per una nazione

- valore osservato: $y_{i}$
- valore atteso: $\lambda_{i}$
- ipotesi nulla: $H_{0} : Y_{i} \sim \text{Poisson}(\lambda_{i})$


> ATTENZIONE, PARTE DA RISCRIVERE. NON SO COME TRADURRE LEFT-TAIL, RIGHT-TAIL
Se ora calcoliamo:

- $P(Y_{i} \leq 5 | \lambda_{i})$ e' significativamente sotto il valore atteso dal punto di vista statistico se il p-value $< \alpha$ (in questo caso $\alpha$ e' il *significance level* (pare in italiano si dica livello di significativita'))
- $P(Y_{i} \geq 5 | \lambda_{i})$ e' significativamente sotto il valore atteso dal punto di vista statistico se il p-value $< \alpha$ (in questo caso $\alpha$ e' il *significance level* (pare in italiano si dica livello di significativita'))

In [7]:
from scipy.stats import poisson

# select Italy's row
row = EU_countries_GDP[EU_countries_GDP['Country'] == 'Italy'].iloc[0]

observed = 5
expected = row["expected_stadiums"]

# compute p-values
p_left = poisson.cdf(observed, expected)          # P(Y ≤ observed)
p_right = 1 - poisson.cdf(observed - 1, expected) # P(Y ≥ observed)
# p_two_sided = 2 * min(p_left, p_right)
# 2023.03.05 Fix: a more proper two-sided p-value for discrete distributions:
p_two_sided = min(1, 2 * min(p_left, p_right))

print("Country:", row["Country"])
print("Expected:", expected)
print("Observed:", observed)
print("p_left:", p_left)
print("p_right:", p_right)
print("p_two_sided:", p_two_sided)

Country: Italy
Expected: 8.446871829845312
Observed: 5
p_left: 0.1536382269624755
p_right: 0.9232515233913667
p_two_sided: 0.307276453924951


Quello che ci interessa, in questo caso, e' il p-value della coda della parte sinistra (il `p_left`) che e' pari a circa 0.15. In questo caso, se scegliamo il solito significant level di 0.05, il p-value e' maggiore del signiicance level $\alpha$ per cui 

> NON possiamo escludere che in realta' il valore di 5 nuovi stadi in Italia, non sia in linea con la media europea e che la differenza con il benchmark che abbiamo trovato nel caso deterministico, puo' essere dovuta a una "fluttuazione statistica" all'interno della variabilita' di ogni nazione dell'EU (Satana). 

#### Livello 2 - casi Europa Dei Popoli (quella vera!!1!!) e UEFA (ladri corrotti e, fondamentalmente, contro la mia squadra qualunque sia la mia squadra). 

Avendo discusso il framework teorico nelle sezioni precedenti, cerchiamo ora di rispondere alla domanda se l'Italia rappresenti veramente il fanalino di coda dell'Europa considerando prima un'Europa Continentale (quella dei popoli sovrani, LE RADICI!!!11!!) e poi l'UEFA (quella delle elite corrotte che ci stanno portando via il calcio vero, quello di una volta quando contavano i sentimenti e non il vile denaro). Il parametro che usiamo e' di nuovo il prodotto interno lordo *pro capite*, quello che cambia e' il numero dei paesi coinvolti. Come anticipato, abbiamo fatto alcune semplificazioni per quanto riguarda l'UEFA e assegnato al GDP di Inghilterra, Galles, Scozia e Irlanda del Nord quello del Regno Unito, senza fare distinzioni tra le diverse regioni, anche se in realta' le diverse zone del Regno Unito hanno un reddito *pro capite* diverso. Nel caso dell'UEFA, quindi, andremo a calcolare il numero di nuovi stadi negli ultimi 20 anni **per federazione** e non per nazione. Se sembriamo pedanti, prendetevela con Il Post che ci costringe a spiegare bene ogni passaggio altrimenti arrivano quelli della redazione con il ditino alzato borbottando "ah ah ah mmmmh"(cit.).  

Per entrambi i casi che riguardano l'Europa Continentale e l'UEFA, calcoliamo prima il valore di benchmark (il valore atteso) e poi la significanza stastistica del discostamento dal valore atteso. 

- Valori attesi

In [8]:
from pathlib import Path

data_path_continental_europe = Path('data/processed/geographic_europe_GDP.pkl')
data_path_UEFA = Path('data/processed/UEFA_countries_GDP.pkl')

geographic_europe_GDP = pd.read_pickle(data_path_continental_europe)
UEFA_countries_GDP = pd.read_pickle(data_path_UEFA)

# total number of stadiums in the last 20 years

TOTAL_STADIUMS = 250

# Compute expected stadiums proportional to GDP per capita for both geographic Europe and UEFA countries

geographic_europe_GDP["GDP_share"] = geographic_europe_GDP["GDP"] / geographic_europe_GDP["GDP"].sum()
geographic_europe_GDP["expected_stadiums"] = TOTAL_STADIUMS * geographic_europe_GDP["GDP_share"]

UEFA_countries_GDP["GDP_share"] = UEFA_countries_GDP["GDP"] / UEFA_countries_GDP["GDP"].sum()
UEFA_countries_GDP["expected_stadiums"] = TOTAL_STADIUMS * UEFA_countries_GDP["GDP_share"]

print(geographic_europe_GDP.head(3))
print('---')
print(UEFA_countries_GDP.head(3))


   Country  Population            GDP  GDP_share  expected_stadiums
0  Albania         2.80   9730.869219   0.004110           1.027471
1  Andorra         0.08  46812.426101   0.019771           4.942867
2  Armenia         2.80   8159.087210   0.003446           0.861508
---
   Country  Population            GDP  GDP_share  expected_stadiums
0  Albania         2.80   9730.869219   0.003799           0.949681
1  Andorra         0.08  46812.426101   0.018275           4.568642
2  Armenia         2.80   8159.087210   0.003185           0.796283


In [9]:
print(geographic_europe_GDP[geographic_europe_GDP['Country'] == 'Italy'])
print('---')
print(UEFA_countries_GDP[UEFA_countries_GDP['Country'] == 'Italy'])


   Country  Population            GDP  GDP_share  expected_stadiums
24   Italy         59.2  39277.083878   0.016589           4.147219
---
   Country  Population            GDP  GDP_share  expected_stadiums
25   Italy         59.2  39277.083878   0.015333           3.833233


Come nel caso di una distribuzione random equiprobabilistica, appena consideriamo tutti i paesi dell'Europa continentale e UEFA, il valore atteso per gli stadi nuovi scende. Sorprendentemente, il valore atteso, ossia quello che dovremmo aspettarci quando ipotizziamo che la costruizione di nuovi stadi dipenda dal prodotto interno lordo, e' inferiore a quello osservato, che per Marotta e' basso. Se dovessimo fidarci del modello, dovremmo concludere che costruire 5 nuovi stadi significa essere al di sopra della media europea, e quindi altro che fanalini di coda.

Perche' abbiamo scritto "sorprendentemente"? Perche' in effetti il GDP dell'Italia dovrebbe essere superiore alla media europea, e quindi il numero atteso di stadi ce lo aspettiamo piu' alto. Vediamo se i dati ci danno un'idea del perche' otteniamo questi risultati.  

In [10]:
geographic_europe_GDP.sort_values(by="expected_stadiums", ascending=False)[:10]

,Country,Population,GDP,GDP_share,expected_stadiums
33,Monaco,0.04,256799.788613,0.108461,27.115176
28,Liechtenstein,0.04,206780.590353,0.087335,21.833710
30,Luxembourg,0.70,133230.619179,0.056271,14.067658
23,Ireland,5.10,106818.917131,0.045116,11.278879
48,Switzerland,8.70,100623.549627,0.042499,10.624718
19,Gibraltar,0.03,100603.000000,0.042490,10.622548
37,Norway,5.50,87497.217965,0.036955,9.238724
22,Iceland,0.40,82138.789297,0.034692,8.672934
14,Faroe Islands,0.05,72465.376966,0.030606,7.651531
12,Denmark,5.90,68043.546697,0.028739,7.184635


E in effetti, come i piu' attenti lettori avranno ipotizzato, ci sono diversi piccoli paesi, quali il principato di Monaco, Liechtenstein, Lussemburgo, Gibilterra, Isole Faroe, hanno un numero atteso di stadi del tutto irragionevole dovuto al loro alto prodotto interno lordo *pro capite*, se non altro considerando la loro estensione e popolazione. Questo, dal punto di vista statistico, allarga a dismisura la distribuzione e comprime verso il basso il valore di aspettazione di nazioni piu' popolose.

Era un effetto facilmente preventivabile, che ogni bravo analista puo' prevedere in anticipo. In questo articolo, lo abbiamo lasciato fino a questo punto anche per sottolineare uno dei punti principali di questa discussione: piu' le dichiarazioni sono vaghe, poco circostanziate, piu' i risultati sono inaffidabili. Questo vale anche quando si sceglie un modello per descrivere la realta' (e prevedere possibili scenari). Quando si sceglie un modello, bisogna essere coscienti del fatto che un modello, per sua natura, ha dei limiti e rappresenta un'approssimazione della realta'. Quando si fanno questo tipo di analisi, si cerca sempre di favorire il modello "piu' economico", ossia quel modello che cerca di tenere in considerazione meno variabili possibili ma che siano quelle che pesano di piu' nella valutazione finale. La ragione per la quale si favoriscono i modelli "piu' economici" e' non solo che computazionalmente piu' leggeri e facilmente maneggiabili (possono essere debuggati piu' facilmente), ma anche perche' quando si devono "spiegare", sono facilmente comprensibili. Trovare l'equilibrio tra la capacita' di modellare bene un fenomeno in maniera e, al tempo stesso, usare il modello per spiegare un fenomeno, e' piu' difficile quanto piu' complicato e' il fenomeno che vogliamo spiegare. Nel caso e del numero di nuovi stadi per paese, ci possono essere cosi' tante variabili che trovare un modello che lo "spieghi" e' difficile. 

In questo caso, poi, e' la natura stessa della questione, con dati confusi e riferimenti poco chiari, a far si che trovare un modello per validare un'affermazione sia molto difficile.

Tornando al nostro modello, solo usando un approccio deterministico per trovare il banchmark, ci dice che la frase di Marotta non regge dato che abbiamo 2 valori, 4.15 e 3.83, che sono al di sotto della soglia, per cui calcolare i p-value e' inutile. 

Quello che possiamo fare, per cercare di dar ragione a Marotta prima dell'arrivo di una querela, e' quello di eliminare dal nostro dataset, tutti i paesi la cui popolazione e' al di sotto del milione di abitanti. E' ovvio che cosi' facendo perdiamo precisione (e' del tutto possibile che paesi ricchi ma con pochi abitanti decidano di costruirsi 10 stadi da 20.000 persone senza chiederci il permesso), ma almeno cerchiamo di giustificarlo dicendo che facciamo un confronto tra realta' un po' piu' omogenee. Avremmo poi potuto costruire un modello che oltre al GDP *pro capite* tenesse in contro altri fattori, ma questo e' uno step che richiederebbe la ricerca di nuovi dati e, dato che lo scopo ultimo di questo articolo e' fondamentalmente il cazzeggio, quindi magari lo faremo se qualcuno offre almeno un caffe'.

Procediamo, quindi, ad eliminare tutti i paesi con popolazione al di sotto del milione di abitanti (ciao ciao Islanda. Grazie comunque per i dischi di Björk). 

In [11]:
geographic_europe_GDP.columns

Index(['Country', 'Population ', 'GDP', 'GDP_share', 'expected_stadiums'], dtype='object')

In [12]:
geographic_europe_GDP.sort_values(by = 'Population ', ascending= True)[:11]

,Country,Population,GDP,GDP_share,expected_stadiums
19,Gibraltar,0.03,100603.000000,0.042490,10.622548
42,San Marino,0.03,59879.717302,0.025291,6.322626
33,Monaco,0.04,256799.788613,0.108461,27.115176
28,Liechtenstein,0.04,206780.590353,0.087335,21.833710
14,Faroe Islands,0.05,72465.376966,0.030606,7.651531
1,Andorra,0.08,46812.426101,0.019771,4.942867
22,Iceland,0.40,82138.789297,0.034692,8.672934
31,Malta,0.50,40905.805959,0.017277,4.319194
34,Montenegro,0.60,12259.877503,0.005178,1.294505
30,Luxembourg,0.70,133230.619179,0.056271,14.067658


In [13]:
UEFA_countries_GDP.sort_values(by = 'Population ', ascending= True)[:11]

,Country,Population,GDP,GDP_share,expected_stadiums
19,Gibraltar,0.03,100603.000000,0.039273,9.818313
44,San Marino,0.03,59879.717302,0.023376,5.843939
34,Monaco,0.04,256799.788613,0.100249,25.062282
29,Liechtenstein,0.04,206780.590353,0.080723,20.180676
14,Faroe Islands,0.05,72465.376966,0.028289,7.072232
1,Andorra,0.08,46812.426101,0.018275,4.568642
22,Iceland,0.40,82138.789297,0.032065,8.016305
32,Malta,0.50,40905.805959,0.015969,3.992187
35,Montenegro,0.60,12259.877503,0.004786,1.196498
31,Luxembourg,0.70,133230.619179,0.052010,13.002594


Giusto per chiarirci, stiamo eliminando nazioni che, dal punto di vista calcistico, sono superiori all'Italia quali Montenegro, Islanda, Lussemburgo e Isole Faroe!

In [14]:
geographic_europe_GDP_filtered = geographic_europe_GDP[geographic_europe_GDP['Population '] >= 1.0][['Country', 
                                                                                                     'Population ',
                                                                                                     'GDP']].copy()

UEFA_countries_GDP_filtered = UEFA_countries_GDP[UEFA_countries_GDP['Population '] >= 1.0][['Country',
                                                                                            'Population ',
                                                                                            'GDP']].copy()

print(geographic_europe_GDP_filtered.shape[0])
print(UEFA_countries_GDP_filtered.shape[0])

42
45


Notiamo subito che il numero di nazioni considerate e' sceso rispettivamente a 42 e 45. Andiamo a calcolare i valori di banchmark usando il GDP.

In [15]:
# total number of stadiums in the last 20 years

TOTAL_STADIUMS = 250

# Compute expected stadiums proportional to GDP per capita for both geographic Europe and UEFA countries

geographic_europe_GDP_filtered["GDP_share"] = geographic_europe_GDP_filtered["GDP"] / geographic_europe_GDP_filtered["GDP"].sum()
geographic_europe_GDP_filtered["expected_stadiums"] = TOTAL_STADIUMS * geographic_europe_GDP_filtered["GDP_share"]

UEFA_countries_GDP_filtered["GDP_share"] = UEFA_countries_GDP_filtered["GDP"] / UEFA_countries_GDP_filtered["GDP"].sum()
UEFA_countries_GDP_filtered["expected_stadiums"] = TOTAL_STADIUMS * UEFA_countries_GDP_filtered["GDP_share"]

print(geographic_europe_GDP_filtered.head(3))
print('---')
print(UEFA_countries_GDP_filtered.head(3))


   Country  Population            GDP  GDP_share  expected_stadiums
0  Albania          2.8   9730.869219   0.007177           1.794304
2  Armenia          2.8   8159.087210   0.006018           1.504479
3  Austria          9.1  56579.504175   0.041731          10.432864
---
   Country  Population            GDP  GDP_share  expected_stadiums
0  Albania          2.8   9730.869219   0.006279           1.569758
2  Armenia          2.8   8159.087210   0.005265           1.316202
3  Austria          9.1  56579.504175   0.036509           9.127256


In [16]:
print(geographic_europe_GDP_filtered[geographic_europe_GDP_filtered['Country'] == 'Italy'])
print('---')
print(UEFA_countries_GDP_filtered[UEFA_countries_GDP_filtered['Country'] == 'Italy'])

   Country  Population            GDP  GDP_share  expected_stadiums
24   Italy         59.2  39277.083878    0.02897           7.242419
---
   Country  Population            GDP  GDP_share  expected_stadiums
25   Italy         59.2  39277.083878   0.025344           6.336075


E finalmente siamo riusciti a far uscire almeno il valore di benchmark per l'Italia superiore al valore atteso per cui Marotta potrebbe avere ragione (aspettiamo a chiamare l'avvocato)! Vediamo ora se la differenza tra valore osservato (5) e valore atteso (rispettivamente 7.24 e 6.33) e' statisticamente significativa.

- CASO EUROPA CONTINENTALE (I POPOLI SOVRANI!!!11!!!)

In [17]:
from scipy.stats import poisson

# select Italy's row
row = geographic_europe_GDP_filtered[geographic_europe_GDP_filtered['Country'] == 'Italy'].iloc[0]

observed = 5
expected = row["expected_stadiums"]

# compute p-values
p_left = poisson.cdf(observed, expected)          # P(Y ≤ observed)
p_right = 1 - poisson.cdf(observed - 1, expected) # P(Y ≥ observed)
# p_two_sided = 2 * min(p_left, p_right)
# 2023.03.05 Fix: a more proper two-sided p-value for discrete distributions:
p_two_sided = min(1, 2 * min(p_left, p_right))

print("Country:", row["Country"])
print("Expected:", expected)
print("Observed:", observed)
print("p_left:", p_left)
print("p_right:", p_right)
print("p_two_sided:", p_two_sided)

Country: Italy
Expected: 7.242419235310879
Observed: 5
p_left: 0.2708240443958881
p_right: 0.84799726898727
p_two_sided: 0.5416480887917762


Il p-value che ci interessa (Numero di stadi inferiore rispetto al valore aspettato) e' il `p_left` pari a circa 0.27. Se usiamo il tipico valore per rigettare l'ipotesi nulla di $\alpha = 0.05$, falliamo nel rigettare l'ipotesi nulla (non c'e' differenza tra valore osservato e valore atteso) e quindi i poteri forti ancora vincono e la frase di Marotta che l'Italia e' un fanalino di coda, non ha una conferma statistica.

Dato che ormai ho scritto il codice, andiamo a calcolarci il p-value anche nel caso considerando i paesi UEFA (sappiamo gia' che a questo punto Marotta e' al telefono con il suo avvocato).

In [18]:
from scipy.stats import poisson

# select Italy's row
row = UEFA_countries_GDP_filtered[UEFA_countries_GDP_filtered['Country'] == 'Italy'].iloc[0]

observed = 5
expected = row["expected_stadiums"]

# compute p-values
p_left = poisson.cdf(observed, expected)          # P(Y ≤ observed)
p_right = 1 - poisson.cdf(observed - 1, expected) # P(Y ≥ observed)
# p_two_sided = 2 * min(p_left, p_right)
# 2023.03.05 Fix: a more proper two-sided p-value for discrete distributions:
p_two_sided = min(1, 2 * min(p_left, p_right))

print("Country:", row["Country"])
print("Expected:", expected)
print("Observed:", observed)
print("p_left:", p_left)
print("p_right:", p_right)
print("p_two_sided:", p_two_sided)

Country: Italy
Expected: 6.3360753950828235
Observed: 5
p_left: 0.39331352208327613
p_right: 0.7574158379009788
p_two_sided: 0.7866270441665523


Peggio che andar di notte. Un p-value di 0.39. Ora, io so che ci sono una montagna di statistici bravi che dicono "eh, ma non dobbiamo essere schiavi del p-value significativo con $\alpha = 0.05$, possiamo benissimo trovare effetti con un $\alpha$ maggiore. Ma 0.39. Io, signor Marotta, ce la metto tutta. Ma 0.39. 

Vabbe, potremmo andare avanti a eliminare altre nazioni che ci stanno antipatiche fino a dimostrare che Marotta ha statisticamente ragione. Ma magari lo facciamo in un'altra occasione dopo aver discusso il Livello 3 (si, c'e' un livello 3. Quello, forse, delle conclusioni). 

### Livello 3: Chiediamolo all'AI

E' difficile non sentire un senso di inadeguatezza, a questo punto, nel poter facilmente dare ragione a Marotta, perche' sicuramente ed evidentemente ha ragione lui. Tanto evidentemente, che non possiamo far altro che arrenderci e, a questo punto, chiedere all'autorita' suprema che e' l'Intelligenza Artificiale. 

Ora, non e' che non ci fidiamo di Marotta, ma per capire dove stiamo sbagliando, ci chiediamo se magari non abbiamo sentito male la sua dichiarazione - oppure e' stata riportata male, ma tenderemmo ad escluderlo data la nota affidabilita' dei giornalisti italiani.

Chiediamo quindi a ChatGPT il seguente prompt:

> create a table with the number of soccer stadiums that have been built in UEFA nations in the last 20 years. Consider stadiums >= 20000 sits. Use stadiumdb.com as source

In [19]:
stadiumdb_data = pd.read_csv('./data/stadiumdb.csv')
stadiumdb_data.head()

,Country (UEFA member),Approx. number of stadiums ≥ 20k seats built 2006–2026,Examples & evidence
0,Albania,2,"Air Albania Stadium (Arena Kombëtare), Tirana ..."
1,Andorra,0,No stadium ≥20k.
2,Armenia,0,No stadium ≥20k.
3,Austria,1,Wörthersee Stadion (28 Black Arena) in Klagenf...
4,Azerbaijan,1,"Baku Olympic Stadium, Baku – capacity ~69,870 ..."


In [20]:
stadiumdb_data['Approx. number of stadiums ≥ 20k seats built 2006–2026'].sum()

np.int64(67)

In [21]:
stadiumdb_data[stadiumdb_data['Country (UEFA member)'] == 'Italy']

,Country (UEFA member),Approx. number of stadiums ≥ 20k seats built 2006–2026,Examples & evidence
26,Italy,2,"Allianz Stadium (Juventus Stadium) – 41,507 se..."


E gia' qui c'e' un piccolo problema, dato che, secondo stadiumdb il numero di stadi in europa non e' di 250 ma 67. E anche il numero di stadi costruito in Italia scende da 5 a 2. 

Usiamo lo stesso prompt per Claude e vediamo se abbiamo gli stessi risultati.

In [22]:
stadiumdb_data_claude = pd.read_csv('./data/uefa_stadiums_2005_2025_claude.csv')
stadiumdb_data_claude.head()

,Country,Stadium,City,Capacity,Opened
0,Albania,Air Albania Stadium,Tirana,22500,2019
1,Austria,Allianz Stadion,Vienna,28345,2016
2,Austria,Red Bull Arena Salzburg (rebuilt),Salzburg,29520,2007
3,Azerbaijan,Baku Olympic Stadium,Baku,68700,2015
4,Belarus,Nacyjanalny Futbolny Stadion,Minsk,33145,2025


In [23]:
stadiumdb_data_claude_summary = (stadiumdb_data_claude
    .groupby('Country', as_index=False)
    .agg({
        'Stadium': ['count', lambda x: ', '.join(x.unique())]
    })
)

stadiumdb_data_claude_summary.columns = ['Country', 'n_stadium', 'stadium_names']
stadiumdb_data_claude_summary

,Country,n_stadium,stadium_names
0,Albania,1,Air Albania Stadium
1,Austria,2,"Allianz Stadion, Red Bull Arena Salzburg (rebu..."
2,Azerbaijan,1,Baku Olympic Stadium
3,Belarus,1,Nacyjanalny Futbolny Stadion
4,Belgium,2,"Ghelamco Arena (KAA Gent), Lotto Park (rebuilt)"
5,Cyprus,1,GSP Stadium (rebuilt)
6,Czech Republic,1,Generali Česká pojišťovna Arena (rebuilt)
7,Denmark,2,"Brøndby Stadion (rebuilt), Ceres Park"
8,England,7,"Amex Stadium, Emirates Stadium, Hill Dickinson..."
9,France,5,"Nouveau Stade de Bordeaux (Matmut Atlantique),..."


In [24]:
(stadiumdb_data[stadiumdb_data['Approx. number of stadiums ≥ 20k seats built 2006–2026'] > 0]
 .reset_index(drop=True)
 .rename(columns={'Country (UEFA member)': 'Country', 'Approx. number of stadiums ≥ 20k seats built 2006–2026': 'n_stadium', 'Examples': 'stadium_names'})
)

,Country,n_stadium,Examples & evidence
0,Albania,2,"Air Albania Stadium (Arena Kombëtare), Tirana ..."
1,Austria,1,Wörthersee Stadion (28 Black Arena) in Klagenf...
2,Azerbaijan,1,"Baku Olympic Stadium, Baku – capacity ~69,870 ..."
3,Belgium,1,"Ghelamco Arena, Ghent – about 20k seats, opene..."
4,Bulgaria,1,"Hristo Botev Stadium, Plovdiv – about 20k seat..."
5,Czech Republic,1,"Eden Arena (Sinobo Stadium) in Prague – ~20,80..."
6,England,5,"Emirates Stadium (60k seats, opened July 2006)..."
7,France,5,"Allianz Riviera (Nice) – 35,624 seats, inaugur..."
8,Georgia,1,"Batumi Stadium (Adjarabet Arena) – 20,035 seat..."
9,Germany,3,"PreZero Arena – 30,150 seats, inaugurated 31 J..."


Con il medesimo prompt, otteniamo due liste differenti. La cosa non ci sorprende piu' di tanto. ChatGPT e Claude possono eseguire diverse strategie per estrarre dati (anche dalla stessa fonte), usare le API per interrogare il sito in maniera differente. Inoltre, per quanto riguarda Russia e Turchia, ChatGPT in realta' aveva dato un numero approssimativo (circa 10) che successivamente e' stato arrotondato (da noi) a 10.  

In [25]:
(stadiumdb_data[stadiumdb_data['Approx. number of stadiums ≥ 20k seats built 2006–2026'] > 0]
 .reset_index(drop=True)
 .rename(columns={'Country (UEFA member)': 'Country', 'Approx. number of stadiums ≥ 20k seats built 2006–2026': 'n_stadium', 'Examples': 'stadium_names'})
).sort_values(by='Country')[:5]

,Country,n_stadium,Examples & evidence
0,Albania,2,"Air Albania Stadium (Arena Kombëtare), Tirana ..."
1,Austria,1,Wörthersee Stadion (28 Black Arena) in Klagenf...
2,Azerbaijan,1,"Baku Olympic Stadium, Baku – capacity ~69,870 ..."
3,Belgium,1,"Ghelamco Arena, Ghent – about 20k seats, opene..."
4,Bulgaria,1,"Hristo Botev Stadium, Plovdiv – about 20k seat..."


In [26]:
stadiumdb_data_claude_summary.sort_values(by='Country')[:5]

,Country,n_stadium,stadium_names
0,Albania,1,Air Albania Stadium
1,Austria,2,"Allianz Stadion, Red Bull Arena Salzburg (rebu..."
2,Azerbaijan,1,Baku Olympic Stadium
3,Belarus,1,Nacyjanalny Futbolny Stadion
4,Belgium,2,"Ghelamco Arena (KAA Gent), Lotto Park (rebuilt)"


In [27]:
stadiumdb_data_claude_summary['n_stadium'].sum()

np.int64(112)

In [28]:
stadiumdb_data_claude_summary[stadiumdb_data_claude_summary['Country'] == 'Italy']

,Country,n_stadium,stadium_names
16,Italy,2,"Juventus Stadium (Allianz Stadium), Stadio Fri..."


In [29]:
stadiumdb_data_chatgpt_summary = (stadiumdb_data[stadiumdb_data['Approx. number of stadiums ≥ 20k seats built 2006–2026'] > 0]
 .reset_index(drop=True)
 .rename(columns={'Country (UEFA member)': 'Country', 
                  'Approx. number of stadiums ≥ 20k seats built 2006–2026': 'n_stadium', 
                  'Examples & evidence': 'stadium_names'})
).copy()

In [30]:
stadiumdb_data_claude_summary[stadiumdb_data_claude_summary['Country'] == 'Italy']['stadium_names'].values

array(['Juventus Stadium (Allianz Stadium), Stadio Friuli (Dacia Arena)'],
      dtype=object)

In [31]:
stadiumdb_data_chatgpt_summary[stadiumdb_data_chatgpt_summary['Country'] == 'Italy']['stadium_names'].values

array(['Allianz Stadium (Juventus Stadium) – 41,507 seats, inaugurated 8 Sep 2011; Dacia Arena in Udine – 25k seats, completed 2016.'],
      dtype=object)

Almeno sul numero di stadi costruiti in Italia, sia ChatGPT che Claude entrambi ne stimano 2: l'Allianz Stadium e la Dacia Arena. Ne mancano ancora 3.

Nuovo prompt:

> You are a skilled Data Analyst. Using stadiumdb.com as a source, list the number and names of stadium built in Italy in the last 20 years without any constraint about the number of seats

Con questo nuovo prompt, Claude, usando [stadiumdb](https://stadiumdb.com/stadiums/ita), mi indica ben 6 stadi!!

| # | Stadium | City | Club(s) | Capacity | Opened |
|---|---------|------|---------|----------|--------|
| 1 | Allianz Stadium | Turin | Juventus FC | 41,507 | 2011 |
| 2 | Bluenergy Stadium (Stadio Friuli) | Udine | Udinese FC | 25,132 | 2016 |
| 3 | New Balance Arena (ex-Gewiss Stadium) | Bergamo | Atalanta BC | 24,950 | 2019–2024 |
| 4 | Unipol Domus (Sardegna Arena) | Cagliari | Cagliari Calcio | 16,412 | 2017 |
| 5 | PSC Arena (Stadio Benito Stirpe) | Frosinone | Frosinone Calcio | 16,227 | 2017 |
| 6 | Stadio Filadelfia | Turin | Torino FC | 4,200 | 2017 |

Mentre ChatGPT rimane fermo a 3!

| # | Stadium | City | Club(s) | Capacity | Opened |
|---|---------|------|---------|----------|--------|
| 1 | Allianz Stadium | Turin | Juventus FC | 41,507 | 2011 |
| 2 | Bluenergy Stadium (Stadio Friuli) | Udine | Udinese FC | 25,132 | 2016 |
| 6 | Stadio Filadelfia | Turin | Torino FC | 4,200 | 2017 |



In [32]:
stadiumdb_data_chatgpt_summary[stadiumdb_data_chatgpt_summary['Country'] == 'England']['stadium_names'].values

array(['Emirates Stadium (60k seats, opened July 2006); Wembley (90k seats, opened March 2007); American Express Stadium (31,876 seats, opened 16 Jul 2011); London Stadium (~60k, reopened 2016); Tottenham Hotspur Stadium (62,850 seats, inaugurated 03 Apr 2019).'],
      dtype=object)

In [33]:
stadiumdb_data_claude_summary[stadiumdb_data_claude_summary['Country'] == 'England']['stadium_names'].values

array(['Amex Stadium, Emirates Stadium, Hill Dickinson Stadium, London Stadium, Stadium MK, Tottenham Hotspur Stadium, Wembley Stadium'],
      dtype=object)

#### Claude da ragione a Marotta. O forse no.

C'e' un aspetto interessante che riguarda la risposta di una dell'AI usate (per la precisione Claude - Sonnet 4.6). Rispondendo al prompt:

> You are a skilled Data Analyst. Using stadiumdb.com as a source, list the number and names of stadium built in Italy in the last 20 years without any constraint about the number of seats

Dopo aver eseguito tutte le sue ricerche e fornito la tabella con i sei stadi riprodotta nel paragrafo precedente, Claude, senza che lo avessimo richiesto, risponde anche:

---

**Total**: 6 confirmed new/fully rebuilt stadiums (2005–2025), plus 1 small community ground with uncertain date.

**Key takeaway**: Italy stands out in Europe for how few new stadiums it has built in the last 20 years. Most of its major grounds — San Siro (1926), Olimpico (1953), San Paolo (1959), San Nicola (1990) — are decades old. This is precisely why, as widely discussed in Italian football, the country faces significant infrastructure challenges ahead of UEFA Euro 2032 [Kiddle](https://kids.kiddle.co/List_of_European_stadiums_by_capacity), which it is co-hosting with Turkey.

---

Quindi, anche secondo Claude, l'Italia sarebbe un'eccezione in Europa in quanto alla costruzione di troppi pochi stadi. Insomma, Claude conferma la tesi di Marotta. A questo punto, dato che e' possibilissimo che Claude possa fare un'analisi molto piu' dettagliata rispetto a quella esposta finora, chiediamo a Claude:

> "Key takeaway: Italy stands out in Europe for how few new stadiums it has built in the last 20 years." - based on what you produce this insight? If you build a probabilistic model with equal probability of new stadium in all countries in Europe, can you infer that Italy stands out? Otherwise how you do to make probabilstic correct statement?

Innanzi tutto, Claude ci risponde:

> You're raising a completely valid methodological point. I stated that insight without any statistical grounding — it was essentially an editorial opinion borrowed from the narrative on StadiumDB news articles, not a conclusion derived from data.

Quindi ci informa che la sua deduzione era basata su *editorial opinion* e da articoli di giornale e non deriva dai dati.

Inoltre mi propone un piano articolato su piu' punti (traduco questa parte in Italiano): 

1. Raccogliere i dati: contare il numero di nuovi stadi nel periodo 2005-2006
2. Normalizzazione: controllare per le [variabili confondenti](https://it.wikipedia.org/wiki/Confondente) prima di confrontare il dato italiano con quello europeo quali:
    - Popolazione
    - Numero di societa' di calcio professionistiche
    - GDP o capacita' di investimento delle infrastrutture
    - Baseline di partenza (quanti stadi vecchi gia' esistevano)
3. Costruzione del modello - ad esempio un modello semplice che confronti il valore atteso con quello misurato. Valore atteso che sia proporzionale a uno delle variabili del punto 2. 
4. Test di significanza

Notiamo che praticamente Claude segue lo stesso workflow che abbiamo seguito finora: raccogliere dati (nel nostro caso li aveva raccolti Marotta per conto nostro); Normalizzare (noi abbiamo prima fatto un modello semplice senza variabili ma con equa probabilita' e poi scelto il GDP pro capite); Costruire un modello (anche Claude ne sceglie uno semplice che sia proporzionale a una delle variabili che ha indicato); Test di significanza (il p-value che abbiamo calcolato)

E infine conclude:

> Without doing any of that, the statement is unsupported. Italy might actually be perfectly average when normalized by club count or GDP. I should not have presented it as a data-driven insight — I'll avoid that kind of ungrounded commentary going forward.

A questo punto, chiediamo a Claude:

> Can you try to perform the analysis you suggest? Give also the p-value

E qui Claude si entusiasma e mi risponde:

> Great challenge. Let me build this properly — I'll gather the data we need first, then run the statistical model.

Quindi Claude scrive:

> Good. Now I have the context I need. Rather than trying to scrape a perfect per-country professional club count (which varies by definition), I'll use top-division clubs as the normalizing variable — a clean, well-defined measure. Let me build the full analysis in Python.

E come risultati da:

> Model 1 — Naive Equal Probability (your suggested baseline)
Every country has the same probability of building any given stadium. Under this model Italy's expected count is 2.14, but it built 6 — giving a two-tailed p-value of 0.045, which is just barely significant at α=0.05. This would technically support the original claim. But this model is poorly specified — it treats Turkey (19 clubs) the same as Montenegro (10 clubs), which is unreasonable.

e poi

> Model 2 — Poisson GLM normalized by top-division club count (the right model)
Once you control for the size of a country's football infrastructure (number of top-division clubs), Italy's expected count jumps to 7.83 — Italy actually built fewer stadiums than expected. The two-tailed p-value is 0.669. Italy is completely unremarkable.
The only genuine outliers are Turkey (built 21, expected ~6, p≈0.000) and Russia (built 13, expected ~3, p≈0.000) — both driven by deliberate national modernization programmes and World Cup/tournament hosting. After Bonferroni correction for multiple comparisons across 49 countries, only Turkey and Russia remain significant.

E qui c'e' un piccolo colpo di scena. Claude, giustamente, nota che Russia e Turchia rappresentano due cosi detti "outlier", ossia 2 valori che spostano di moltoi risultati, dato che hanno costruito molti piu' stadi negli ultimi anni rispetto alle altre nazioni europee (ricordiamo che Claude sta valutando solo il caso UEFA). Con il Modello 2 (quello piu' sofisticato, insomma), il valore atteso per l'Italia sale a 7.83, quello osservato e' di 6, e abbiamo un p-value di 0.7 circa che non ci permette di rigettare l'ipotesi nulla, il che implica che  l'affermazione che l'Italia abbia pochi stadi nuovi rispetto all'Europa **non e'** statisticamente valida.  

Ancora piu' interessante e' che, per quanto riguarda il Modello 1, Claude scrive: 

> Under this model Italy's expected count is 2.14, but it built 6 — giving a two-tailed p-value of 0.045, which is just barely significant at α=0.05. This would technically support the original claim.

Praticamente, quello che Claude afferma, e' che con un valore di aspettazione (o valore atteso) di 2.14 e un valore osservato di 6, abbiamo un p-value di poco al di sotto di $\alpha = 0.05$ che ci permetterebbe di scartare l'ipotesi nulla anche se di poco (questo mettendo nel conto i due outlier). Ma qui Claude commette un errore. Logicamente, se il valore atteso e' di circa 2 stadi nuovi, e l'Italia ne ha costruiti ben 6 (quindi piu' dei 2 che ci aspetterebbe), allora l'affermazione nulla "non esiste differenza tra media Europea e l'Italia rispetto alla costruizione di nuovi stadi" e' si falsa, ma nel senso opposto rispetto al claim che l'Italia ne abbia costruiti di meno. 

Facciamo notare questa cosa a Claude:

> "Every country has the same probability of building any given stadium. Under this model Italy's expected count is 2.14, but it built 6 — giving a two-tailed p-value of 0.045, which is just barely significant at α=0.05. This would technically support the original claim. But this model is poorly specified — it treats Turkey (19 clubs) the same as Montenegro (10 clubs), which is unreasonable.". Can you explain better this? In my understanding, the original claim is " Italy stands out in Europe for how few new stadiums it has built in the last 20 years". The expected value for Italy is 2.14 but it built 6. In this case, if I understood right, the observed value (i.e. number of new stadium) is higher than the expected one (2.14), meaning that actually Italy built more stadium than expected, contrary of the original claim. What am I missing?

(da notare come il prompt parta dall'idea che siamo noi che sbagliamo a dire che lui ha tratto una conlcusione non corretta). Ma Claude, onestamente, risponde:

> You are missing nothing — you are completely right, and I made an error in my interpretation.
Let me be precise:
The original claim was that Italy built few stadiums. Under Model 1, Italy built 6, against an expected value of 2.14. The observed value is above the expected, meaning Italy actually built more than the naive average. So Model 1 does not support the original claim at all — it actually contradicts it.
When I wrote "this would technically support the original claim", that was simply wrong. I confused the direction of the effect. The p-value of 0.045 tells you Italy is a statistically unusual country under Model 1, but the direction of the deviation is upward (too many stadiums), not downward (too few). A significant p-value alone says nothing about direction — you have to look at the residual, which is +3.86 (observed minus expected), pointing in the opposite direction of the claim.
So to summarise cleanly across both models:

| Model | Expected for Italy | Observed | Direction | Supports "Italy built few"? |
| ----- | ------------------ | -------- | ---------- | --------------------------|
| Model 1 (naive) | 2.14  | 6 | Italy built more than average | ❌ NO |
| Model 2 (GLP, normalized) | 7.83 | 6 | Italy built slightly less than expected | Direction yes, but $p = 0.67$, so not significant |

E quindi conclude con:

> The original claim is unsupported under both models. Under Model 1 it is outright contradicted. Under Model 2, Italy is slightly below expectation but nowhere near significantly so. Thank you for catching this — it was a meaningful error in the interpretation.




# Conclusioni

Prima di discutere le conclusioni, vorremmo fare un paio di premesse a scanso di equivoci (e anche per evitare querele).

L'intento principale dell'articolo e' quello di divertirsi: prendere una frase che usa dei dati numerici per affermare un "fatto" e mettere alla prova i dati che porta. Un po' come dire "potrebbe anche sembrarmi sensato, ma e' veramente dimostrabile dal punto di vista statistico?". Nel divertimento, poi, ci piacerebbe anche spiegare che esistono approcci "scientifici" nel dimostrare alcune affermazioni: tutti possiamo (e dobbiamo) avere delle opinioni, quando si presentano come fatti, c'e' il modo di capire se si possono dimostrare o meno. Un altro punto dell'articolo, che poi e' una fissazione personale, e' che statistica e senso comune spesso possano essere in contrasto. Per quanto un'affermazione possa sembrare ragionevole e sensata, la realta', invece, ci dice che ci possano essere altre spiegazioni o altre interpretazioni. 

Nel caso specifico, Marotta puo' avere ragione nel credere che l'Italia sia indietro sulla costruizione di stadi rispetto all'Europa. Il punto e' che quando si fa un'affermazione cosi' vaga come quella fatta, c'e' sempre qualcuno che puo' obiettare che i fatti riportati (se precisi, inoltre) non hanno alcun fondamento nella realta'. 

Quando si fa uno test statistico, ci si deve innanzitutto basare su dati quanto piu' precisi, raccolti con una data metodologia e che, possibilmente, siano reperibili da tutti. Come abbiamo visto, e' difficile reperire i numeri citati, il che non vuol dire che siano falsi o sbagliati, significa che non sappiamo come sono stati raccolti, da chi, e che criteri ha usato per raccoglierli (ammesso che la frase sia riportata bene o che Marotta non si sia confuso). 

Come abbiamo cercato di spiegare nell'articolo, possiamo proporre diversi modelli per supportare un'affermazione e considerare diversi parametri. A seconda delle ipotesi statistiche che si fanno, il modello che li cattura rapprensentera' piu' o meno bene la realta'. Dato che l'affermazione iniziale era vaga, possiamo concludere comunque che usando dei modelli semplici, l'affermazione non regge l'ipotesi statistica: non e' un fatto che l'Italia costruisca meno stadi che in Europa. 

Costruire uno stadio nuovo e' un'opportunita' per tanti e con costi che ricadono, in genere, anche su chi non ha interesse nel calcio. Quando si ospitano eventi quali mondiali, europei, olimpiadi, poi, c'e' un ulteriore impulso, fondamentalmente da parte degli stati ospitanti, che favorisce questo tipo di interventi. L'ultima volta che l'Italia ha avuto questo tipo di opportunita', e' stato nel 1990 e, in quell'occasione, furono costruiti nuovi stadi e ristrutturati molti di quelli esistenti. E' particolare notare come uno dei pochi stadi costruiti negli ultimi 20 anni sia quello che ospita la Juventus. Proprio Juventus e Torino, dal 1990 al 2006, avevano giocato in uno stadio costruito per l'occasione: lo [stadio delle Alpi](https://it.wikipedia.org/wiki/Stadio_delle_Alpi). Quindi, nel 1990 si costruisce un nuovo stadio per poi essere abbattuto nel 2009 - quando l'eta' media degli stadi Italiani e' molto piu' alta. Si potrebbe concludere che quella di Italia '90 sia stata un'occasione sprecata che ha anche generato sospetto, nell'opinione pubblica, un certo sospetto per questo genere di operazioni. E fondamentalmente, in Italia, non si costruisce un nuovo stadio senza la collaborazione o l'aiuto fondamentale delle istituzioni comunali. Ma questa e' un'opinione, non un fatto. Pero' se si dovesse costruire un modello piu' sofisticato di quello presentato andrebbe modellato statisticamente e aggiunto alle altre variabili.

Un altro punto che e' stato trattato e' quello dell'utilizzo delle cosidette AI per fare analisi statistiche e, da quello che si e' scritto, sembrerebbe che possano portare a errori grossolani. Infatti Claude, nelle sue conclusioni, da per certa un'opinione che non e' derivata dai fatti, ma dai commenti che ha, presumibilmente, trovato in alcuni articoli in rete. Quando poi viene sfidata a dare una conlusione statistica, ammette che da questo punto di vista non si tratta di un fatto, anzi. Potremmo anche sottolineare il fatto che Claude, per sua stessa ammissione, sbaglia in maniera clamorosa la conclusione della sua stessa analisi statistica. Saremmo tentati dallo scrivere l'ennesimo commento di quanto queste AI non siano cosi' intelligenti, che fanno errori stupidi etc etc. In realta', i modelli LLM stanno migliorando moltissimo da questo punto di vista e, nel primo periodo nel quale erano state proposte, la situazione era decisamente peggiore, con errori incredibili e allucinazioni grossolane. Le AI sono, allo stato attuale, dei "collaboratori" formidabili: aiutano spesso in task noiosi e ripetitivi nei quali la nostra (genere sapiens) attenzione non regge e commettiamo (noi sapiens) molti piu' errori dei sistemi LLM. Come ogni collaboratore, il dialogo e l'iterazione porta a risultati soddisfacenti, mentre aver fiducia in tutto quello che si viene detto (o quello che si dice) non crediamo sia l'approccio piu' utile.

#### References
- GDP per Capita: [World Bank Group](https://data.worldbank.org/indicator/NY.GDP.PCAP.CD)